src/utils/traffic-mesh.js

```diff
@@ -18,6 +20,8 @@ const installTrafficMesh = async ({ log }) => {
     execArgs: ['--version'],
     pattern: '\\sv(.+)',
     execName: EXEC_NAME,
+    latestVersion: LATEST_VERSION,
+    log,
```

Comment:
The `log` argument was missing. It is used here https://github.com/netlify/cli/pull/1456/files#diff-f8e66f1838601194468460f6e49a5bad17e2a2700afce376413bb7a4a9c8a290R66